sdk/include/reactos/rosctrls.h

```diff
@@ -581,3 +581,254 @@ class CTreeView :
     }
 
 };
+
+class CTooltips :
+    public CWindow
+{
+public: // Configuration methods
+
+    HWND Create(HWND hWndParent, DWORD dwStyles = 0, DWORD dwExStyles = 0)
+    {
+        if (!dwStyles)
+        {
+            dwStyles = WS_POPUP | TTS_NOPREFIX;
+        }
+
+        if (!dwExStyles)
+        {
+            dwExStyles = WS_EX_TOPMOST;
+        }
+
+        m_hWnd = CreateWindowExW(dwExStyles,
+            TOOLTIPS_CLASS,
+            TEXT(""),
+            dwStyles,
+            CW_USEDEFAULT, CW_USEDEFAULT,
+            CW_USEDEFAULT, CW_USEDEFAULT,
+            hWndParent,
+            NULL,
+            _AtlBaseModule.GetModuleInstance(),
+            NULL);
+
+        if (!m_hWnd)
+            return NULL;
+
+        SetWindowPos(HWND_TOPMOST, 0, 0, 0, 0, SWP_NOMOVE | SWP_NOSIZE | SWP_NOACTIVATE);
+
+        return m_hWnd;
+    }
+
+public: // Relay event
+
+        // Win7+: Can use GetMessageExtraInfo to provide the WPARAM value.
+    VOID RelayEvent(MSG * pMsg, WPARAM extraInfo = 0)
+    {
+        SendMessageW(TTM_RELAYEVENT, extraInfo, reinterpret_cast<LPARAM>(pMsg));
```

Comment:
Use SendMessage, otherwise this most likely won't compile against MS's ATL.

Reply:
It does. I wrote that in the standalone app. ;P